Re-enable GOV.UK Content API docs job for AWS Jenkins

kevindew opened the pull request:
This re-enables the job which got lost in the switch to AWS and switches the
frequency the job runs at from once an hour to once a day.

kevindew commented:
Unfortunately this fails due to bundle install permissions. I'll look into it when I grab some time.

issyl0 commented:
I just rebased this on master and resolved the conflicts. I wonder if this is mergeable or still fails on `bundle install`.

kevindew commented:
@issyl0 Thanks for that

I just tried again and same problem:

```
Cloning into 'content-store'...
npm install
/var/lib/jenkins/workspace/govuk-content-api-docs/govuk-content-api-docs
└─┬ widdershins@2.1.4  (git+https://github.com/alphagov/widdershins.git#b4b723a8ca10318892a4f5d1087d1e09604a664d)
  ├── dot@1.1.2 
  ├── jgexml@0.3.8 
  ├─┬ js-yaml@3.12.0 
  │ ├─┬ argparse@1.0.10 
  │ │ └── sprintf-js@1.0.3 
  │ └── esprima@4.0.1 
  ├── openapi-sampler@0.3.3 
  ├─┬ openapi_optimise@1.1.0 
  │ ├── lodash@4.17.10 
  │ ├── reftools@0.0.20 
  │ └─┬ yargs@11.1.0 
  │   ├─┬ cliui@4.1.0 
  │   │ └─┬ strip-ansi@4.0.0 
  │   │   └── ansi-regex@3.0.0 
  │   ├─┬ find-up@2.1.0 
  │   │ └─┬ locate-path@2.0.0 
  │   │   ├─┬ p-locate@2.0.0 
  │   │   │ └─┬ p-limit@1.3.0 
  │   │   │   └── p-try@1.0.0 
  │   │   └── path-exists@3.0.0 
  │   ├─┬ os-locale@2.1.0 
  │   │ ├─┬ execa@0.7.0 
  │   │ │ ├─┬ cross-spawn@5.1.0 
  │   │ │ │ ├─┬ lru-cache@4.1.3 
  │   │ │ │ │ ├── pseudomap@1.0.2 
  │   │ │ │ │ └── yallist@2.1.2 
  │   │ │ │ ├─┬ shebang-command@1.2.0 
  │   │ │ │ │ └── shebang-regex@1.0.0 
  │   │ │ │ └─┬ which@1.3.1 
  │   │ │ │   └── isexe@2.0.0 
  │   │ │ ├── get-stream@3.0.0 
  │   │ │ ├── is-stream@1.1.0 
  │   │ │ ├─┬ npm-run-path@2.0.2 
  │   │ │ │ └── path-key@2.0.1 
  │   │ │ ├── p-finally@1.0.0 
  │   │ │ ├── signal-exit@3.0.2 
  │   │ │ └── strip-eof@1.0.0 
  │   │ └─┬ mem@1.1.0 
  │   │   └── mimic-fn@1.2.0 
  │   ├─┬ string-width@2.1.1 
  │   │ └── is-fullwidth-code-point@2.0.0 
  │   ├── which-module@2.0.0 
  │   └─┬ yargs-parser@9.0.2 
  │     └── camelcase@4.1.0 
  ├─┬ swagger2openapi@2.11.16 
  │ ├─┬ ajv@5.5.2 
  │ │ ├── fast-deep-equal@1.1.0 
  │ │ ├── fast-json-stable-stringify@2.0.0 
  │ │ └── json-schema-traverse@0.3.1 
  │ ├── call-me-maybe@1.0.1 
  │ ├── co@4.6.0 
  │ ├── node-fetch@2.2.0 
  │ ├─┬ node-readfiles@0.2.0 
  │ │ └── es6-promise@3.3.1 
  │ ├─┬ should@13.2.3 
  │ │ ├── should-equal@2.0.0 
  │ │ ├── should-format@3.0.3 
  │ │ ├── should-type@1.4.0 
  │ │ ├── should-type-adaptors@1.1.0 
  │ │ └── should-util@1.0.0 
  │ └── yargs@11.1.0 
  └─┬ yargs@5.0.0 
    ├─┬ cliui@3.2.0 
    │ ├─┬ strip-ansi@3.0.1 
    │ │ └── ansi-regex@2.1.1 
    │ └─┬ wrap-ansi@2.1.0 
    │   ├─┬ string-width@1.0.2 
    │   │ └── is-fullwidth-code-point@1.0.0 
    │   └─┬ strip-ansi@3.0.1 
    │     └── ansi-regex@2.1.1 
    ├── decamelize@1.2.0 
    ├── get-caller-file@1.0.3 
    ├── lodash.assign@4.2.0 
    ├─┬ os-locale@1.4.0 
    │ └─┬ lcid@1.0.0 
    │   └── invert-kv@1.0.0 
    ├─┬ read-pkg-up@1.0.1 
    │ ├─┬ find-up@1.1.2 
    │ │ ├── path-exists@2.1.0 
    │ │ └─┬ pinkie-promise@2.0.1 
    │ │   └── pinkie@2.0.4 
    │ └─┬ read-pkg@1.1.0 
    │   ├─┬ load-json-file@1.1.0 
    │   │ ├── graceful-fs@4.1.11 
    │   │ ├─┬ parse-json@2.2.0 
    │   │ │ └─┬ error-ex@1.3.2 
    │   │ │   └── is-arrayish@0.2.1 
    │   │ ├── pify@2.3.0 
    │   │ └─┬ strip-bom@2.0.0 
    │   │   └── is-utf8@0.2.1 
    │   ├─┬ normalize-package-data@2.4.0 
    │   │ ├── hosted-git-info@2.7.1 
    │   │ ├─┬ is-builtin-module@1.0.0 
    │   │ │ └── builtin-modules@1.1.1 
    │   │ ├── semver@5.5.0 
    │   │ └─┬ validate-npm-package-license@3.0.4 
    │   │   ├─┬ spdx-correct@3.0.0 
    │   │   │ └── spdx-license-ids@3.0.0 
    │   │   └─┬ spdx-expression-parse@3.0.0 
    │   │     └── spdx-exceptions@2.1.0 
    │   └── path-type@1.1.0 
    ├── require-directory@2.1.1 
    ├── require-main-filename@1.0.1 
    ├── set-blocking@2.0.0 
    ├─┬ string-width@1.0.2 
    │ ├── code-point-at@1.1.0 
    │ └─┬ is-fullwidth-code-point@1.0.0 
    │   └── number-is-nan@1.0.1 
    ├── which-module@1.0.0 
    ├── window-size@0.2.0 
    ├── y18n@3.2.1 
    └─┬ yargs-parser@3.2.0 
      └── camelcase@3.0.0 

npm WARN govuk-content-api-docs No repository field.
npm WARN govuk-content-api-docs No license field.
bundle install
sudo: no tty present and no askpass program specified
Bundler requires sudo access to install at the moment. Try installing again,
granting Bundler sudo access when prompted, or installing into a different path.
make: *** [requirements] Error 30
Build step 'Execute shell' marked build as failure
Finished: FAILURE
```

It's pretty weird really as surely there are a bunch of places where these jobs do bundle installs

kevindew commented:
@issyl0 I did some digging and found the reason why it was failing (lack of directory existence for bundler). I've added a command to create it now and it all seems good.

Also rebased again.
